sample/core.py

```diff
@@ -108,9 +109,10 @@ def _append_results(currlist, newlist, prop, strip_chars):
 
 if __name__ == "__main__":
     titles = ['pantera', 'opeth']
-
+    with open('metadata.json') as data_file:
+        data = json.load(data_file)
     try:
-        wk = WikiPage(titles=titles)
+        wk = WikiPage(data, titles=titles)
```

Comment:
@abhigyank Don't do it in main. Do it inside `__init__`. Sending the metadata from WikiPage instantiation is bad design.